src/wasm/wasm-binary.cpp

```diff
@@ -534,22 +528,18 @@ void WasmBinaryWriter::writeEvents() {
 }
 
 void WasmBinaryWriter::writeNames() {
-  bool hasContents = false;
-  if (wasm->functions.size() > 0) {
-    hasContents = true;
-    getFunctionIndex(wasm->functions[0]->name); // generate mappedFunctions
```

Comment:
`getFunctionIndex` does not generate things (it asserts if the name exists), so I guess this comment may not be not have been up-to-date